### PR TITLE
Made predictions for solar flare, added linting

### PR DESCRIPTION
**Pull request by Rohith04MVK**

### CODE REVIEW

**Comment by Rohith04MVK** [comment]
I have linted everything

File: src/solar-flare/main.py [hunk 1]
```diff
@@ -1,4 +1,11 @@
 import pandas as pd
+from sklearn.model_selection import train_test_split
+from sklearn.multioutput import MultiOutputClassifier
+from sklearn.preprocessing import LabelEncoder
+from sklearn.svm import SVC
+
+
```
Comment (janaSunrise) [inline src/solar-flare/main.py:7]:
```suggestion

```

Change it to one line :)

File: src/solar-flare/main.py [hunk 2]
```diff
@@ -7,7 +14,19 @@
...
+flare_2 = pd.read_csv("flare2.csv", sep=" ", names=column_names).apply(LabelEncoder().fit_transform)
 
 df = pd.concat([flare_1, flare_2])
+
+X = df.drop(["common_flare", "moderate_flare", "severe_flare"], axis=1)
+y = df[["common_flare", "moderate_flare", "severe_flare"]]
+
+X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.2, random_state=42)
+
+
```
Comment (janaSunrise) [inline src/solar-flare/main.py:27]:
```suggestion

```

Same here ;)

**Review by janaSunrise — CHANGES_REQUESTED** [review]
Do the changes to be approved 👍🏼

**Review by janaSunrise — APPROVED** [review]
Great! 👍🏼

**Pull request merged**